controllers/operatorpolicy_controller.go

```diff
@@ -594,10 +595,14 @@ func (r *OperatorPolicyReconciler) applySubscriptionDefaults(
 	)
 	if err != nil {
 		if k8serrors.IsNotFound(err) {
-			return fmt.Errorf(
-				"%wthe subscription defaults could not be determined because the PackageManifest was not found",
-				ErrPackageManifest,
-			)
+			if !r.usingExistingSubIfFound(policy, subscription) {
```

Comment:
We don't have any status change for usingExistingSubIfFound

Reply:
It doesn't make a special status directly, it just allows the policy to be considered valid so that the status of the subscription can be updated (an invalid policy doesn't update the other conditions). The subscription status will have more information in some cases, for example explaining that the operator is not available in a catalog.